app/scripts.babel/element_observer.js

```diff
@@ -0,0 +1,41 @@
+const HANDLER_CLASS_PREFIX = 'handler_';
```

Comment:
I suggest something like `tf__observer-123` so that it's easier to know where the class comes from.